content/blog/2015/06/2015-06-29-first-unstable-daily-iso/index.md

```diff
@@ -44,16 +44,16 @@ qemu-system-x86_64 -enable-kvm -m 4096M -cdrom Solus-Daily.iso
 
 ### Other Awesome Updates
 
-Our [Patreon](https://www.patreon.com/solus?ty=h) campaign is really starting to take shape! We recently hit our first target of $150 a month, which will cover base project running costs. This is a monthly project sponsorship, and helps us to do all the 
```

Comment:
I think these can be left as is, as they just show the inactive Patreon page, which is fine IMO.

Alternatively they could be replaced by a link to the OpenCollective, but that's for extra credit.